src/tim/vx/ops/conv2d.cc

```diff
@@ -34,9 +34,10 @@ namespace ops {
 Conv2d::Conv2d(Graph* graph, int32_t weights, PadType padding,
                const std::array<uint32_t, 2>& ksize,
                const std::array<uint32_t, 2>& stride,
-               const std::array<uint32_t, 2>& dilation, int32_t multiplier)
-    : Conv2d(graph, weights, padding, ksize, stride, dilation, {0, 0, 0, 0},
-             multiplier) {}
+               const std::array<uint32_t, 2>& dilation,
+               int32_t multiplier)
```

Comment:
once we added data_layout as a parameter for conv2d, we should not allow user create Conv2d without data layout parameter. I think we can add layout with default value for Conv2D constructor.

Reply:
Ok, so we need not add new constructor for operations.